usagov/usagov-2021
# USAGOV-1686-Give-images-explicit-dims: fixed improper spanish header …

…usagov logo


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1718

## Description

Fixed the bug in the spanish usagov logo, added the right height dimensions ins the _header.scss file
## Type of Changes

- [ x] Bugfix
  - Add screenshot showing what it should look like

## Testing Instructions



localhost > espanol page > resize window big and small to make sure the usagov logo is properly sized and still responsive at different viewport sizes

### Requires New Config
- [ x] No

### Requires New Content
- [x ] No

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
